openedx/features/pakx/lms/pakx_admin_app/serializers.py

```diff
@@ -14,18 +16,30 @@ class UserSerializer(serializers.ModelSerializer):
     employee_id = serializers.CharField(source='profile.employee_id')
     language = serializers.CharField(source='profile.language')
     name = serializers.CharField(source='get_full_name')
+    organization = serializers.CharField(source='profile.organization')
     role = serializers.SerializerMethodField()
 
     class Meta:
         model = User
-        fields = ('id', 'email', 'name', 'employee_id', 'language', 'is_active', 'role')
+        fields = ('username', 'id', 'email', 'name', 'employee_id', 'organization', 'language', 'is_active', 'role', 'date_joined', 'last_login')
```

Comment:
why we have `organization` here? 
Can we have a separate serializer for the retrieve User API instead of using listAPI one?